doc/Tex/RAM_SCB_testing.tex

```diff
@@ -1,77 +1,57 @@
-  reference solutions stored in the output directory of the installation 
-  location.  If the two files have values that differ by a certain amount 
-  (default is an absolute difference of $10^{-30}$), the test will fail.}
+\item{Create a run directory entitled \verb*lrun_testl.  If one exists, it will be deleted, so use caution when using this name for any other purpose besides automated testing.}
+\item{Copy the required input and parameter files into the new test run directory.}
+\item{Run the code. Test simulations are only long enough to perform the features being tested.}
+\item{Perform a specialized version of \verb*ldiffl, included in the distribution, to compare the results produced by the test simulation against reference solutions stored in the output directory of the installation location. If the two files have values that differ by a certain amount (default is an absolute difference of $10^{-9}$), the test will fail.}
 \end{enumerate}
 
-Each of these steps can be called individually by using {\tt make}
-\textit{nametest}\_\textit{stepname}, where the step names are {\tt compile},
-{\tt rundir}, {\tt run}, and {\tt check}.
+Each of these steps can be called individually by using {\tt make} \textit{nametest}\_\textit{stepname}, where the step names are {\tt compile}, {\tt rundir}, {\tt run}, and {\tt check}.
 
-If, at any point, the test fails, the details will be recorded in 
-\textit{nametest}.diff in the installation directory.  If the test is 
-successful, this file will be created but be empty.  Tests that fail at the 
-comparitive stage will list all of the differences.  When this happens, 
-additional tools to examine the file differences are recommended.  An 
-excellent visualization of file differences can be rendered with 
-\verb*ltkdiffl, an open source tool that is already installed on many 
-Linux machines.
+If, at any point, the test fails, the details will be recorded in \textit{nametest}.diff in the installation directory. If the test is successful, this file will be created but be empty. Tests that fail at the comparitive stage will list all of the differences. When this happens, additional tools to examine the file differences are recommended. An excellent visualization of file differences can be rendered with \verb*ltkdiffl, an open source tool that is already installed on many Linux machines.
 
 \section{Description of Available Tests}
 
 \subsection{Test 1}
 
 \textbf{Command:} \verb*lmakel \verb*ltest1l
 
-This is the most basic test for RAM-SCB.  It runs RAM-SCB for 300 seconds using
-Volland-Stern electric field, constant dipole magnetic field (no SCB 
-calculation), and LANL fluxes.  \textbf{SHIELDS-RC} users should rely on this
-test.
+This is the most basic test for RAM-SCB. It runs RAM-SCB for 900 seconds using Volland-Stern electric field, constant dipole magnetic field (no SCB calculation), and LANL fluxes. \textbf{SHIELDS-RC} users should rely on this test.
```

Comment:
Do we want discussion of SHIELDS-RC here? Is that mode still a directive in the PARAM?
If we've nixed it then we need to clean up discussion of SHIELDS-RC...

Reply:
It is still there, and still sets the appropriate flags